DER/DerivativesContracts/Options.rdf

```diff
@@ -651,11 +984,6 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The option premium is the income received by the seller (writer) of an option contract to another party. In-the-money option premiums are composed of two factors: intrinsic and extrinsic value. Out-of-the-money options&apos; premiums consist solely of extrinsic value. For stock options, the premium is quoted as a dollar amount per share, and most contracts represent the commitment of 100 shares.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
I don't understand this split - it does not have any reality. Whether an option is in the money could change between it being offered for sale and actually purchased